yaml/places.yaml

```diff
@@ -39,6 +39,42 @@ global:
         - [ 9, 9 ]
       default: 10
 
+  - &ne_country_capital [Admin-0 region capital, Admin-0 capital alt, Admin-0 capital]
```

Comment:
Nit: format the same as next row?

`Admin-0 capital, Admin-0 capital alt, Admin-0 region capital`

Reply:
Fixed in 04c96b34.